### ISSUE TITLE
Not opening links

### ISSUE THREAD

**Issue by Gophertv** [clearyy/discord-link-opener#15]
In the console it says opened: and whatever the link is but it doesn't actualy open the link, how do i fix this, im running chrome beta, is that a problem?

**Comment by ImDini** [comment]
> In the console it says opened: and whatever the link is but it doesn't actualy open the link, how do i fix this, im running chrome beta, is that a problem?

Yes that is a problem. Edit open.py should be around line 37. You need to change the path to your browser location.
